service/src/main/java/uk/gov/hmcts/reform/fpl/config/utils/LookupConfigParser.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.fpl.config.utils;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
+
+@SuppressWarnings({"PMD", "checkstyle:hideutilityclassconstructor"})
```

Comment:
now then 🙄 can you not fix the checkstyle error? and whats the mistake from the PMD?

Reply:
forgot this was here. just wants a private constructor, added.

Reply:
Agree we should hide constructor. Please do

```
private LookupConfigParser() {
  // NO-OP
}
```

Sorry forgotten about it.